projects/plugins/jetpack/extensions/shared/memberships/settings.scss

```diff
@@ -15,7 +15,8 @@
 .jetpack-newsletter-access-radio-buttons {
 	label {
 		text-wrap: balance;
-		max-width: 100%;
+		flex-shrink: 0;
+		margin-right: 12px;
```

Comment:
Not sure we need those styles. I would keep it consistent with Categories and remove this additional margin. What is the purpose of `flex-shrink`? When I apply it, Chrome tells me it's doing nothing but maybe I'm doing sth wrong 🤷‍♂️ 

<img width="480" alt="Screenshot 2024-02-19 at 22 19 17" src="https://github.com/Automattic/jetpack/assets/4068554/0db5327d-e91e-44ea-9f38-053dac8beee0">

Apart from that it looks good, thanks!

Reply:
Noticed sth weird. When the text is longer it looks like this:

<img width="321" alt="Screenshot 2024-02-19 at 22 48 54" src="https://github.com/Automattic/jetpack/assets/4068554/938a697d-8763-4da5-a3ab-9e9f2a6c49fc">

Reply:
I guess `flex-shrink: 0` should be applied to the checkbox to prevent it from shrinking.

```css
.jetpack-newsletter-access-radio-buttons {
	input[type=radio] {
		flex-shrink: 0;
	}

	label {
		text-wrap: balance;
	}
}
```